corehq/warehouse/models/staging.py

```diff
@@ -183,8 +183,9 @@ class UserStagingTable(StagingTable, CouchToDjangoETLMixin):
     email = models.CharField(max_length=255, null=True)
     doc_type = models.CharField(max_length=100)
     base_doc = models.CharField(max_length=100)
-    domain = models.CharField(max_length=100)
+    domain = models.CharField(max_length=100, null=True, blank=True)
 
+    domain_memberships = JSONField(null=True)
```

Comment:
This will be JSOn only in staging table, it can be unpacked in the domainmembership_dim.sql (which I haven't committed here yet)